EventsExpress.Core/Services/LocationService.cs

```diff
@@ -41,23 +41,35 @@ public async Task<Guid> Create(LocationDto locationDTO)
             return result.Id;
         }
 
-        public LocationDto LocationByPoint(Point point)
+        public Guid? LocationByPoint(Point point)
         {
-            var locationDTO = Mapper.Map<LocationDto>(Context.EventLocations
+            var locationDto = Mapper.Map<LocationDto>(Context.Locations
                 .Where(e =>
                     e.Point.X == point.X &&
                     e.Point.Y == point.Y)
                 .FirstOrDefault());
-            return locationDTO;
+            return locationDto.Id;
         }
 
-        public LocationDto LocationByURI(string uri)
+        public Guid? LocationByURI(string uri)
         {
-            var locationDTO = Mapper.Map<LocationDto>(Context.EventLocations
+            var locationDTO = Mapper.Map<LocationDto>(Context.Locations
                 .Where(e =>
                    e.OnlineMeeting == uri)
                 .FirstOrDefault());
-            return locationDTO;
+            return locationDTO.Id;
+        }
+
+        public async Task<Guid> AddLocationToUser(LocationDto locationDto)
```

Comment:
overall LGTM, but @iuriikhrystiuk, I want to discuss contexts boundaries here. I guess we can move validation out of location service since this belongs to the user (or event) domain. If so, we can simply have one function `Task<Guid> AddLocation(LocationDto locationDto)`, and perform validation higher in the call hierarchy. 

Still think we need a better name for this since it isn't gonna call SaveChanges() so not 'Add'... Probably can be Generate, or Build... don't know, just thinking out of loud...